VERSION

```diff
@@ -1 +1 @@
-0.10.7
\ No newline at end of file
+0.10.8-alpha.1
```

Comment:
I think that this should be dev-1 instead of alpha.1

Reply:
I was debating that.  Updated